documentation/asciidoc/computers/compute-module/introduction.adoc

```diff
@@ -33,36 +40,67 @@ The previous form factor could not have supported these interfaces.
 
 === Compute Module 3
 
 .Compute Module 3
 image::images/cm3.jpg[alt="Compute Module 3", width="60%"]
 
-The Compute Module 3 (CM3) contains the internals of a Raspberry Pi 3 (the BCM2837 processor and 1GB of RAM) as well as an optional 4GB of eMMC flash storage.
+The Compute Module 3 (CM3) combines the internals of a Raspberry Pi 3 (the BCM2837 processor and 1GB of RAM) with an optional 4GB of eMMC flash storage. CM3 comes in the DDR2 SO-DIMM form factor.
 
 === Compute Module 1
 
 .Compute Module 1
 image::images/cm1.jpg[alt="Compute Module 1", width="60%"]
 
-The Compute Module 1 (CM1) contains the internals of a Raspberry Pi (the BCM2835 processor and 512MB of RAM) as well as an optional 4GB of eMMC flash storage.
+The Compute Module 1 (CM1) contains the internals of a Raspberry Pi (the BCM2835 processor and 512MB of RAM) as well as an optional 4GB of eMMC flash storage. CM1 comes in the DDR2 SO-DIMM form factor.
 
 == IO Boards
 
-Raspberry Pi IO Boards provide a way to connect a single Compute Module to a variety of I/O (input/output) interfaces. Compute Modules are, by nature, small. As a result, they lack ports and connectors. IO Boards provide a way to connect Compute Modules to a variety of peripherals.
+Raspberry Pi IO Boards provide a way to connect a single Compute Module to a variety of I/O (input/output) interfaces. Compute Modules are small, lacking ports and connectors. IO Boards provide a way to connect Compute Modules to a variety of peripherals.
+
+Raspberry Pi IO Boards provide the following functionality:
+
+* power the module
+* connects the GPIO to pin headers
+* connects the camera and display interfaces to FFC connectors
+* connects HDMI to an HDMI port
+* connects USB to USB ports
+* connects activity monitoring to LEDs
+* eMMC programming over USB
+* connects PCIe to connectors used to physically connect storage or peripherals
 
 IO Boards are breakout boards intended for development; in production, you should use a smaller, potentially custom board that provides only the ports and peripherals required for your use-case.
 
+=== Compute Module 5 IO Board
+
+.Compute Module 5 IO Board
+image::images/cm5io.png[alt="Compute Module 5 IO Board", width="60%"]
+
+Compute Module 5 IO Board provides the following interfaces:
+
+* HAT footprint with 40-pin GPIO connector
+* PoE header
+* 2× HDMI ports
+* 2× USB 3.0 ports
+* Gigabit Ethernet RJ45 with PoE support
+* M.2 M key PCIe socket compatible with the 2230, 2242, 2260, and 2280 form factors
+* microSD card slot (only for use with Lite variants with no eMMC; other variants ignore the slot)
+* 2× MIPI DSI/CSI-2 combined display/camera FPC connectors (22-pin 0.5 mm pitch cable)
+* Real-time clock with battery socket
+* four-pin JST-SH PWM fan connector
+* 5V @ 5A power input via USB-C using the same standard as Raspberry Pi 5
+* Jumpers to disable features such as eMMC boot, EEPROM write, the power button, and the USB OTG connection
```

Comment:
I've not read the datasheet yet, but does the jumper actually disable the power button, or does it just allow you to connect an additional external power button?